examples/diff_capri.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+
+# execute this script from inside the `examples/` folder.
+
+# HADDOCK main relative path
+HM="../../haddock3main/examples"
+
+diff_capri () {
```

Comment:
Why is this being done with shell and not with python? Can't we parse these and compare if the values are within a tolerable range? How can the developer tell if the numbers "match"?

Reply:
The quick answer is that I just wanted to share my scripts with the main dev team. Since we are not yet on an official release. I can make it on Python, but for testing, is that needed?

Regarding comparing the values within a tolerable range. Do we want a tolerable range for integration tests? Remember tests are run on the same machines and the numbers should be equal.

The developer knows the numbers match because `diff` gives no output. Otherwise `diff` shows the different lines.

Reply:
> I can make it on Python, but for testing, is that needed?

"The project guidelines" say haddock3 is python exclusive (: 

> Remember tests are run on the same machines and the numbers should be equal.

So maybe I did not understand what this part is doing, is it not comparing values obtained by the tests with an expected one?

Reply:
Okay, better make it in python. It might be useful for the future.

> So maybe I did not understand what this part is doing, is it not comparing values obtained by the tests with an expected one?

No, is just using the bash `diff` command. If files are equal, no output. If numbers are different, files are different, `diff` outputs the difference. Just that.